toolbox

```diff
@@ -477,6 +478,105 @@ list_containers()
 )
 
 
+run_rm()
+(
+    force_remove=$1
+    command_to_run=$2
+    id_or_name=$3
+    keyword=$4
+
+    if $force_remove; then
+        if ! rm_output=$($prefix_sudo podman $command_to_run $id_or_name --force 2>&42); then
+            echo "$0: failed to remove $id_or_name"
+            exit 1
+        fi
+    else
+        if ! rm_output=$($prefix_sudo podman $command_to_run $id_or_name 2>&42); then
+            echo "$0: failed to remove $id_or_name"
+            exit 1
+        fi
+    fi
+    if ! [ "$rm_output" = "" ]; then
+        echo -e "${LRC}$keyword $rm_output removed${NC}"
+    fi
+)
+
+
+remove_image()
+(
+    id_or_name_list=$1
+    force_remove=$2
+    remove_all=$3
+
+    if ! $remove_all; then
+        IFS=' '
+        export IFS
+        for id_or_name in $id_or_name_list; do
```

Comment:
POSIX doesn't support arrays other than the ones for command line parameters (ie., `$1`, `$2`, etc.). We can use a `while` loop with `read` and newline separated elements to work around that.

Reply:
Done.